libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-row-delete.service.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  ElementRef,
+  InjectionToken,
+  Signal,
+  WritableSignal,
+} from '@angular/core';
+
+import { GridApi } from 'ag-grid-community';
+import { Subject, Subscription } from 'rxjs';
+
+// Use a token because the constructor parameters are not injectable.
+export const SKY_AG_GRID_ROW_DELETE_SERVICE =
+  new InjectionToken<SkyAgGridRowDeleteService>('SkyAgGridRowDeleteService');
+
+/**
+ * Used to provide dependencies to the `SkyAgGridRowDeleteComponent` overlay.
+ * @internal
+ */
+export class SkyAgGridRowDeleteService {
```

Comment:
We usually reserve the "service" keyword for injectables. What about `SkyAgGridRowDeleteContext`, to mimic the same behavior we have with modals?

Reply:
Makes sense. Updated.